helm/grafana/templates/cnpg/access-keys-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and (.Values.postgresqlCluster.enabled) (.Values.postgresqlCluster.customAccessKeys.use) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "grafana-app.labels" . | nindent 4 }}
+  name: {{ .Values.postgresqlCluster.name }}-access-keys
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+stringData:
+  ACCESS_KEY_ID: "{{ .Values.postgresqlCluster.customAccessKeys.accessKeyId }}"
```

Comment:
This looks like it is only for s3.
I would have this under postgresqlCluster.bucket.s3.accessKeyId and also add postgresqlCluster.type.s3?